datasafe-simple-adapter/datasafe-simple-adapter-spring/src/main/java/de/adorsys/datasafe/simple/adapter/spring/annotations/UseDatasafeSpringConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package de.adorsys.datasafe.simple.adapter.spring.annotations;
+
+import de.adorsys.datasafe.simple.adapter.spring.DatasafeSpringConfiguration;
+import org.springframework.context.annotation.Import;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Retention(value = java.lang.annotation.RetentionPolicy.RUNTIME)
+@Target(value = {java.lang.annotation.ElementType.TYPE})
```

Comment:
Can we import all stuff statically?

Reply:
? i do it always this way. never thought about....